command-line/command-line-interface.md

```diff
@@ -37,6 +37,7 @@ $ cd Downloads/
 
 ### Supplemental Materials
 
+- [An introduction to bash](https://guide.bash.academy/inception/).This is a complete guide to bash. It explain its installation and need.It will help you to start using bash. It is reccomended that you should go through this guide after completing the lesson.  
```

Comment:
reccomended -> recommended
Add whitespace after full stop.
(.This) -> (. This)
(.It) -> (. It)